test/test_ops.py

```diff
@@ -999,6 +1015,14 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             expected = torch.tensor([[1.0, 0.9933, 0.9673], [0.9933, 1.0, 0.9737], [0.9673, 0.9737, 1.0]])
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
+    def test_giou_jit(self):
+        box_tensor = torch.tensor([[0, 0, 100, 100], [0, 0, 50, 50], [200, 200, 300, 300]], dtype=torch.float)
```

Comment:
I remember adding these and calculating by hand an year ago. Great that they are still used! :smile: